bin/migrate-oats-data/batch_applications.py

```diff
@@ -188,6 +132,15 @@ def clean_applications(conn=None):
         )
         print(f"Deleted items count = {cursor.rowcount}")
 
+        cursor.execute(
```

Comment:
I do not think you need to drop these tables after the import. They could work as a good reference to why some of the applications were excluded.